### PR TITLE
HFP-3780 flashcards screen reader confused by card transitions

### PR DESCRIPTION
**Pull request by runarkasp**
- Remove duplicate aria labels picked up by screen reader

### CODE REVIEW

File: js/flashcards.js [hunk 1]
```diff
@@ -568,28 +570,34 @@ H5P.Flashcards = (function ($, XapiGenerator) {
...
     // Remove from existing card.
     if (this.$current) {
       this.$current.find('.h5p-textinput').attr('tabindex', '-1');
       this.$current.find('.joubel-tip-container').attr('tabindex', '-1');
       this.$current.find('.h5p-check-button').attr('tabindex', '-1');
       this.$current.find('.h5p-icon-button').attr('tabindex', '-1');
     }
-
-    // Set new card
+
```
Comment (otacke) [inline js/flashcards.js:583]: You might want to configure your IDE to trim extras spaces automatically ...

File: js/flashcards.js [hunk 2]
```diff
@@ -568,28 +570,34 @@ H5P.Flashcards = (function ($, XapiGenerator) {
...
     if (this.$current) {
       this.$current.find('.h5p-textinput').attr('tabindex', '-1');
       this.$current.find('.joubel-tip-container').attr('tabindex', '-1');
       this.$current.find('.h5p-check-button').attr('tabindex', '-1');
       this.$current.find('.h5p-icon-button').attr('tabindex', '-1');
     }
-
-    // Set new card
+    
+    // Set new card
```
Comment (otacke) [inline js/flashcards.js:584]: Same here.

File: js/flashcards.js [hunk 3]
```diff
@@ -568,28 +570,34 @@ H5P.Flashcards = (function ($, XapiGenerator) {
...
-    $card.one('transitionend', function () {
-      if ($card.hasClass('h5p-current') && !$card.find('.h5p-textinput')[0].disabled) {
-        $card.find('.h5p-textinput').focus();
-      }
-      setTimeout(function () {
-        this.announceCurrentPage();
-      }.bind(this), 500);
-    }.bind(this));
+       
+       // Handle focus shift and screen reader announcement
```
Comment (otacke) [inline js/flashcards.js:591]: Your IDE (if configures properly) should also be able to tell you that the indentation is odd here.

File: js/flashcards.js [hunk 4]
```diff
@@ -612,7 +620,7 @@ H5P.Flashcards = (function ($, XapiGenerator) {
     $card.find('.h5p-check-button').attr('tabindex', '0');
     $card.find('.h5p-icon-button').attr('tabindex', '0');
     $card.find('.joubel-tip-container').attr('tabindex', '0');
-  };
+};
```
Comment (otacke) [inline js/flashcards.js:623]: I think it was correct before.

File: js/flashcards.js [hunk 5]
```diff
@@ -621,7 +629,7 @@ H5P.Flashcards = (function ($, XapiGenerator) {
     const pageText = this.options.pageAnnouncement
       .replace('@current', this.$current.index() + 1)
       .replace('@total', this.options.cards.length.toString());
-    this.$pageAnnouncer.text(pageText);
+      this.$pageAnnouncer.text(pageText);
```
Comment (otacke) [inline js/flashcards.js:632]: Was also correct before.

**Review by otacke — COMMENTED** [review]
For your consideration :-)

**Comment by runarkasp** [comment]
Thanks, I forgot to look properly through before pushing. Will address the wrongdoings :)

**Pull request closed without merging**